### PR TITLE
chore: update top picks file 03-10-2023

### PR DESCRIPTION
**Pull request by taddes**
## References

JIRA: NA

## Description
Update Top Picks 03-10-2023

## PR Review Checklist

_Put an `x` in the boxes that apply_

- [x] This PR conforms to the [Contribution Guidelines](https://github.com/mozilla-services/merino-py/blob/main/CONTRIBUTING.md)
- [x] The PR title starts with the JIRA issue reference, format `[DISCO-####]`, and has the same title (if applicable)
- [ ] `[load test: (abort|warn)]` keywords are applied (if applicable)
- [ ] [Documentation](https://github.com/mozilla-services/merino-py/tree/main/docs) has been updated (if applicable)
- [ ] [Functional and performance test](https://github.com/mozilla-services/merino-py/blob/main/docs/dev/testing.md) coverage has been expanded and maintained (if applicable)

### CODE REVIEW

File: dev/top_picks.json [hunk 1]
```diff
@@ -5153,187 +5187,179 @@
...
-                "Video Streaming",
-                "Gaming"
+                "Forums",
+                "Music"
             ],
-            "url": "https://www.jeuxvideo.com",
-            "title": "jeuxvideo.com toute l'actualit\u00e9 du jeu vid\u00e9o et du hardware",
-            "icon": "https://merino-images.services.mozilla.com/favicons/1c01f50c2c503e5557fcb28f6df421bd88640ceba3747d6146f0266fb44700ec_6254.png"
+            "url": "https://rateyourmusic.com",
+            "title": "IP blocked",
```
Comment (pdehaan) [inline dev/top_picks.json:5283]: IP blocker

File: dev/top_picks.json [hunk 2]
```diff
@@ -5355,254 +5381,263 @@
...
+            "rank": 528,
             "domain": "pcpartpicker",
             "categories": [
                 "Personal Blogs",
                 "Technology"
             ],
             "url": "https://pcpartpicker.com",
-            "title": "Pick parts. Build your PC. Compare and share. - PCPartPicker",
-            "icon": "https://merino-images.services.mozilla.com/favicons/8bb92eeedbeff5b5c5a123a7c85bed1680776c98e197eb4196c9b00003774cf3_140345.png"
+            "title": "Unavailable",
```
Comment (pdehaan) [inline dev/top_picks.json:5391]: Revert back to `"Pick parts. Build your PC. Compare and share. - PCPartPicker",`?

File: dev/top_picks.json [hunk 3]
```diff
@@ -6024,39 +6070,49 @@
...
         {
-            "rank": 600,
+            "rank": 601,
+            "domain": "joann",
+            "categories": [
+                "Arts & Crafts",
+                "Ecommerce"
+            ],
+            "url": "https://www.joann.com",
+            "title": "Are You A Bot?",
```
Comment (pdehaan) [inline dev/top_picks.json:6111]: Captcha page?

File: dev/top_picks.json [hunk 4]
```diff
@@ -7474,7 +7645,48 @@
...
+            "title": "Real Estate | Homes for Sale & Rent by Point2",
+            "icon": "https://merino-images.services.mozilla.com/favicons/075e14f30e2305e167303ed9d8d79dc5b0a0e5ef2688b0ee0e773cf95c78261c_9469.png"
+        },
+        {
+            "rank": 757,
+            "domain": "winloot",
+            "categories": [
+                "Shopping"
+            ],
+            "url": "https://winloot.com",
```
Comment (pdehaan) [inline dev/top_picks.json:7664]: Looks like an ad?

**Comment by taddes** [comment]
Thanks so much for checking in on this while @Trinaa is out @pdehaan ! As we've discussed on slack, will be rolling out some additional tooling to handle the whack-a-mole nature of titles and such getting swapped out and changed. I'll follow up when comparing some of the results and will add some of the new blocking strings to the job so it just outputs the domain title. The bad thing is in the last 4 files, I always see instances of AI, IPS, rate limiters and the like making the titles inaccurate.  We'll find a balanced approach that doesn't involve too much heaving lifting on our end to ensure more consistent results.

File: dev/top_picks.json [hunk 5]
```diff
@@ -2429,28 +2353,38 @@
...
+            "domain": "rt",
             "categories": [
-                "Business",
-                "Economy & Finance"
+                "News & Media"
             ],
-            "url": "https://squareup.com",
-            "title": "Square | Solutions For Your Small, Medium & Large Business",
-            "icon": ""
+            "url": "https://www.rt.com",
```
[the remark this answers is not in the document]
Comment (taddes) [reply dev/top_picks.json:2361]: Yeah... good point!

**Comment by taddes** [comment]
@pdehaan I've updated tittles and will take some of the sensitive domains you recommended to the team. We can include them in the new blocklist feature in the provider backend, then they should not be indexed in the future. Thanks again so much!

**Comment by pdehaan** [comment]
Ran my differ through <kbd>grep</kbd> and found these potentially unexpected `title` changes:

```diff
npm start | grep -A1 "\-  title:" | grep -v "\-  url:"

-  title: "Sam's Club - Wholesale Prices on Top Brands"
+  title: "Samsclub"
--
-  title: "Sports Illustrated"
+  title: "si.com"
--
-  title: "Stock Market - Business News, Market Data, Stock Analysis - TheStreet"
+  title: "thestreet.com"
--
-  title: "Parade: Entertainment, Recipes, Health, Life, Holidays"
+  title: "parade.com"
--
-  title: "Alaska Airlines - Flight Deals and Cheap Airline Tickets - Book Today"
+  title: "Alaskaair"
--
-  title: "Government Surplus Auctions - govdeals.com"
+  title: "Home"
```

File: dev/top_picks.json [hunk 6]
```diff
@@ -7070,27 +7234,17 @@
...
+            "rank": 714,
+            "domain": "govdeals",
             "categories": [
-                "Travel"
+                "Ecommerce"
             ],
-            "url": "https://www.westjet.com",
-            "title": "Flights and vacation packages | WestJet official site",
-            "icon": "https://merino-images.services.mozilla.com/favicons/42817e21669e0fc975d246d919a0bb198b3c4bbae8f71083166f800bea59fdcd_34494.ico"
+            "url": "https://beta.govdeals.com",
```
Comment (pdehaan) [inline dev/top_picks.json:7242]: Weird shift from `www.` to `beta.` subdomain. Not sure if we got A-B tested? 🤷 

```diff
 {
-  title: "Government Surplus Auctions - govdeals.com"
+  title: "Home"
-  url: "https://www.govdeals.com"
+  url: "https://beta.govdeals.com"
-  rank: 698
+  rank: 714
 }
```

**Review by pdehaan — APPROVED** [review]
LGTM, but I didn't go through and verify each domain, so there may be tiny nits elsewhere.
For example, not sure how publishers clearing house is a top site, but hey:
```sh
npm run summary | grep "pch"

[375] "Free Online Sweepstakes & Contests | PCH.com" -- https://www.pch.com
```

**Comment by taddes** [comment]
Thanks @pdehaan , I think a lot of those that had longer titles got flagged by bot blockers and as a countermeasure, if we have some enumerated block strings that return, the domain is returned just like that. I'll give it a glance further and make whatever other little changes need to be done. We will for sure be in touch in having a good approach with the automated process to generate significant diffs. I will keep you in the loop as we move on with that and show you the feature for your review too. 🚀

**Review by mtbottle — APPROVED** [review]

**Pull request merged**